packages/loot-core/src/server/accounts/parse-file.js

```diff
@@ -85,7 +87,106 @@ async function parseQIF(filepath) {
   };
 }
 
+/// converts from YYYYMMDDHHMMSS.XXX[gmt offset:tz name]
+/// example: 20230407000000[-5:EST]
+/// to a JS Date object.
+/// Also supports 20190123120000 (YYYYMMDDHHMMSS) format,
+/// assumes local time.
+function ofxDateParse(raw) {
+  const longRx =
+    /(\d{4})(\d{2})(\d{2})(\d{2})(\d{2})(\d{2})(\.\d{3})?\[(.+):(.+)\]/;
+  const shortRx = /(\d{4})(\d{2})(\d{2})(\d{2})(\d{2})(\d{2})/;
+  let res = longRx.exec(raw);
+  if (res) {
+    const [_, y, m, day, hr, min, sec, _milli, offsetStr, _tz] = res;
+    const dateLocal = new Date(y, m - 1, day, hr, min, sec);
+    const offset = parseInt(offsetStr, 10);
+    const diff = offset + dateLocal.getTimezoneOffset();
+    const date = new Date(dateLocal.getTime() + diff);
+    return dateFormat(date, 'yyyy-MM-dd');
+  } else {
+    res = shortRx.exec(raw);
+    if (res) {
+      const [_, y, m, day, hr, min, sec] = res;
+      const date = new Date(y, m - 1, day, hr, min, sec);
+      return dateFormat(date, 'yyyy-MM-dd');
+    }
+  }
+  console.warn('Not able to parse date', raw);
+  return null;
+}
+
 async function parseOFX(filepath) {
+  let objectEq = (arg1, arg2) => {
+    if (
+      Object.prototype.toString.call(arg1) ===
+      Object.prototype.toString.call(arg2)
+    ) {
+      if (
+        Object.prototype.toString.call(arg1) === '[object Object]' ||
+        Object.prototype.toString.call(arg1) === '[object Array]'
+      ) {
+        if (Object.keys(arg1).length !== Object.keys(arg2).length) {
+          return false;
+        }
+        return Object.keys(arg1).every(function (key) {
+          return objectEq(arg1[key], arg2[key]);
+        });
+      } else if (typeof arg1 === 'number' && typeof arg2 === 'number') {
+        return Math.abs(arg1 - arg2) < Number.EPSILON;
+      }
+      return arg1 === arg2;
+    } else if (
+      (arg1 === undefined || arg1 === null) &&
+      (arg2 === undefined || arg2 === null)
+    ) {
+      return true;
+    }
+    return false;
+  };
+  const oldData = await parseOfxNodeLibofx(filepath);
+  const newData = await parseOfxJavascript(filepath);
+  if (!objectEq(oldData, newData)) {
+    console.log('OFX importers are not equal.');
+    console.log('OLD: ', oldData);
+    console.log('NEW: ', newData);
+  }
+  return oldData;
```

Comment:
In case of difference, could you add a `newData` property to `oldData`? Then, in `ImportTransactions.js` please pop open a notification with a message to report an issue and attach a redacted copy of the offending file (if you add an `addNotification` prop, it should be set up automatically). Please keep the console logs so that we can get people to help narrow down the issue if the redaction process obscures it.